plugins/push/frontend/public/javascripts/countly.models.js

```diff
@@ -1548,12 +1659,153 @@
                     });
             });
         },
+        fetchAll: function(type) {
```

Comment:
The service will use the api object to make the calls. It's responsbility is only to know what to do with data transfer objects.